instance-scheduler/main_test.go

```diff
@@ -303,10 +323,119 @@ func TestStopStartTestInstancesInMemberAccount(t *testing.T) {
 			action:        "Test",
 			expectedCount: InstanceCount{4, 3, 2},
 		},
+		{
```

Comment:
because your function is not decoupled, you cannot test ec2 and rds clients seperately. this clause suggestsa new test case